graph/client.py

```diff
@@ -79,16 +105,37 @@ def prepare_graphs(self, rules):
             raise ValueError('Rule: "{}" Error: "{}"'.format(rule, error))
 
     def open_web_browser(self):
-        try:
-            webbrowser.get('firefox').open_new_tab(
-                'html_interpreter/index.html')
-        except BaseException:
-            webbrowser.open_new_tab('html_interpreter/index.html')
+        if not self.off_webbrowser:
+            try:
+                webbrowser.get('firefox').open_new_tab(
+                    'html_interpreter/index.html')
+            except BaseException:
+                webbrowser.open_new_tab('html_interpreter/index.html')
 
     def parse_arguments(self, args):
         parser = argparse.ArgumentParser(
-            description='Client for visualization of SCAP rule evaluation results')
-
+            description='Client for visualization scanned rule from Security scan.')
+        parser.add_argument(
+            '--showFailRules',
```

Comment:
Using Camel Cases is unusual in command line options. I recommend something like `--show-fail-rules`